dev/CommonStyles/FlyoutPresenter_themeresources.xaml

```diff
@@ -54,13 +49,12 @@
             <Setter.Value>
                 <ControlTemplate TargetType="FlyoutPresenter">
                     <Border
-                        Background="{TemplateBinding Background}"
-                        contract7Present:BackgroundSizing="OuterBorderEdge"
+                        Background="{TemplateBinding Background}"                        
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                        contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"
```

Comment:
Wondering why this is removed?

Reply:
It's a bug. We don't have corner radius before win7